spec/support/common_helpers.rb

```diff
@@ -150,10 +150,10 @@ def transition(machine, state_event, user)
   # Closes a modal dialog and waits until it's really closed
   #
   def close_modal_dialog
-    within(".modal-diaglog") do
+    within(".modal") do
       click_link "Back"
     end
-    page.should_not have_css(".modal-backdrop", :wait => 30)
+    page.should_not have_css(".modal-backdrop", :wait => 15)
```

Comment:
Since this got introduced in the same PR, it would be nice to remove it completely from the commit history. In other words, to squash it into 372efc494ffc0a6aba55d75fad1bd578277775db.

That's a bit advanced and might go wrong the first time. So I would recommend to make a copy of your branch first.

The best way to revert this change might be:

```

git rebase -i 372efc494ffc0a6aba55d75fad1bd578277775db^
# Change the first line (the one for 372efc494ffc) from pick xyz to edit xyz
# Once you closed the editor checked out the state of 372efc494ffc
# Edit spec/support/common_helpers.rb
git add spec/support/common_helpers.rb
git rebase --continue
# solve any merge conflicts
```

[gitread.com](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html) also explains this nicely. They have a nice overview of and explanations for various git commands.